### DISCUSSION TITLE
Make db file password protected

### DISCUSSION THREAD

**Discussion by MustafaEminn** [capacitor-community/sqlite#489] [Q&A] [answered]
Hello, I want to make the db file password protected. I do not want to encrypt whole db. I just need a sqlite password layer to connect db. If I will handle this on client side then someone can view db content easily.

**Comment by jepiqueau** [comment, marked as answer]
@MustafaEminn if the db is password protected, the database is encrypted. if you do not encrypt the database put a password to the application or use the biometrics
